PatientRegistrationApp/src/components/HouseholdHomeAddress.vue

```diff
@@ -96,120 +99,97 @@ import EventBus from '../eventBus'
 	data () {
 		return {
 			postalCodeRules: [
-        (v) => !!v || "Zip code is required",
-			(v) =>
-			/(^\d{5}$)|(^\d{5}-\d{4}$)/.test(v) ||
-			"Zip code must be in format of ##### or #####-####",
-		],
+				(v) => !!v || "Zip code is required",
+				(v) =>
+				/(^\d{5}$)|(^\d{5}-\d{4}$)/.test(v) ||
+				"Zip code must be in format of ##### or #####-####",
+				],
 		
-			state:[
-		'AL', 'AK', 'AS', 'AZ',
-		'AR', 'CA', 'CO', 'CT',
-		'DE', 'DC', 'FM',
-		'FL', 'GA', 'GU', 'HI', 'ID',
-		'IL', 'IN', 'IA', 'KS', 'KY',
-		'LA', 'ME', 'MH', 'MD',
-		'MA', 'MI', 'MN', 'MS',
-		'MO', 'MT', 'NE', 'NV',
-		'NH', 'NJ', 'NM', 'NY',
-		'NC', 'ND', 'MP', 'OH',
-		'OK', 'OR', 'PW', 'PA', 'PR',
-		'RI', 'SC', 'SD', 'TN',
-		'TX', 'UT', 'VT', 'VI', 'VA',
-		'WA', 'WV', 'WI', 'WY',
-		],
+			stateOptions:[
+				'AL', 'AK', 'AS', 'AZ',
+				'AR', 'CA', 'CO', 'CT',
+				'DE', 'DC', 'FM',
+				'FL', 'GA', 'GU', 'HI', 'ID',
+				'IL', 'IN', 'IA', 'KS', 'KY',
+				'LA', 'ME', 'MH', 'MD',
+				'MA', 'MI', 'MN', 'MS',
+				'MO', 'MT', 'NE', 'NV',
+				'NH', 'NJ', 'NM', 'NY',
+				'NC', 'ND', 'MP', 'OH',
+				'OK', 'OR', 'PW', 'PA', 'PR',
+				'RI', 'SC', 'SD', 'TN',
+				'TX', 'UT', 'VT', 'VI', 'VA',
+				'WA', 'WV', 'WI', 'WY',
+				],
 			addressTypeOptions: ["Home", "Business", "Temporary"],
-			country: ['USA'],
+			countryOptions: ['USA'],
 			addressType: '',
-			householdLineAddress1: '',
-			householdLineAddress2: '',
-			householdCityAddress: '',
-			householdStateAddress: '',
-			householdCountryAddress: 'USA',
-			householdPostalCode: '',
+			lineAddress1: '',
+			lineAddress2: '',
+			cityAddress: '',
+			stateAddress: '',
+			countryAddress: 'USA',
+			postalCode: '',
 		}
 		
 	},
 	methods: {
-		sendHouseholdHomeAddressInfoToReviewPage()
-		{
+		sendHouseholdHomeAddressInfoToReviewPage() {
 			const householdHomeAddressPayload = {
 				addressType: this.addressType,
-				householdLineAddress1: this.householdLineAddress1,
-				householdLineAddress2: this.householdLineAddress2,
-				householdCityAddress: this.householdCityAddress,
-				householdStateAddress: this.householdStateAddress,
-				householdCountryAddress: this.householdCountryAddress,
-				householdPostalCode: this.householdPostalCode
+				lineAddress1: this.lineAddress1,
+				lineAddress2: this.lineAddress2,
+				cityAddress: this.cityAddress,
+				stateAddress: this.stateAddress,
+				countryAddress: this.countryAddress,
+				postalCode: this.postalCode
 			}
 			EventBus.$emit('DATA_HOUSEHOLD_ADDRESS_INFO_PUBLISHED', householdHomeAddressPayload)
 		},
-		verifyFormContents()
-		{
-			//add logic to check form contents
+		verifyFormContents() {
 			var valid = true
 			var message = "Woops! You need to enter the following field(s):"
 
-			if(this.addressType == "") 
-			{
+			if(this.addressType == "") {
 				message += " Address Type"
 				valid = false
 			}	
-		
-			if(this.householdLineAddress1 == "") 
-			{
-				if(!valid){
+			if(this.lineAddress1 == "") {
```

Comment:
Is there a specific reason why we haven't combined these checks to read?
``` 
if(this.lineAddress1 == "" && !valid) {...}
'''

Reply:
The logic is used to structure the alert message with commas in the appropriate places.  It basically goes like this:
if lineAddress1 is blank {
  if we already started the alert message string (valid == false), insert a comma and space
  add " Address" to the alert message.

Reply:
Wow senior moment or something, yes that makes sense.